src/components/book/book.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-console */
+/* eslint-disable react/destructuring-assignment */
+const Book = (book, key) => {
+  const { id, title, category } = book.obj;
+  return (
+    <tr>
+      <td id={key}>{id}</td>
+      <td>{title}</td>
+      <td>{category}</td>
+    </tr>
+  );
+};
+
+export default Book;
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.

- [ ] <b>REQUIRED:</b> Props you are using in this component are not validated by propTypes. kindly do that.